.buildkite/build.rayci.yml

```diff
@@ -52,3 +52,14 @@ steps:
       - forge
       - ray-mlpy38cu118base
     job_env: forge
+
+  - label: ":tapioca: build: anyscale-ml py38 cu118 docker"
```

Comment:
build here to test that it works; eventually this will be built in release test pipeline only